lighthouse-core/config/config.js

```diff
@@ -145,6 +145,31 @@ function validatePasses(passes, audits, rootPath) {
   });
 }
 
+function validateCategories(categories, audits, auditResults) {
+  if (!categories) {
+    return;
+  }
+
+  if (!audits && !auditResults) {
+    throw new Error('could not find audits or auditResults');
```

Comment:
this feels like total nitpicking, because this is indeed an error state, but this doesn't really feel like validating categories any more. Maybe `validateCategories` should just use an empty array if both don't exist and the fact that both exist should be checked in some overall validator

(currently this error state is caught in runner (and we [test it there](https://github.com/GoogleChrome/lighthouse/blob/9349c894a6a9d4f9b1f0a74ce4555433591d98aa/lighthouse-core/test/runner-test.js#L274)) since we didn't have `Config` back when it was written. It probably should be checked in Config so it never gets that far)

Feel free to disagree :)